.buildkite/commands/save-cache.sh

```diff
@@ -2,40 +2,8 @@
 
 set -euo pipefail
 
-echo "--- :rubygems: Setting up Gems"
-install_gems
-
-echo "--- :closed_lock_with_key: Installing Secrets"
-bundle exec fastlane run configure_apply
-
-# .buildkite/commands/prototype-build.sh -> build_and_upload_prototype_build
-# -> prototype_flavor = 'Jalapeno'
-# -> prototype_build_type = 'Debug'
-echo "--- 🛠 Download Mobile App Dependencies [Assemble Mobile App]"
-./gradlew :WooCommerce:assembleJalapenoDebug
-echo ""
-
-# .buildkite/commands/prototype-build.sh -> build_and_upload_prototype_build
-# -> prototype_flavor = 'Jalapeno'
-# -> prototype_build_type = 'Debug'
-echo "--- 🛠 Download Wear App Dependencies [Assemble Wear App]"
-./gradlew :WooCommerce-Wear:assembleJalapenoDebug
-echo ""
-
-# .buildkite/commands/lint.sh -> ./gradlew :WooCommerce:lintJalapenoDebug
-echo "--- 🧹 Download Lint Dependencies [Lint Mobile App]"
-./gradlew :WooCommerce:lintJalapenoDebug
-echo ""
-
-# .buildkite/commands/run-unit-tests.sh -> ./gradlew testJalapenoDebugUnitTest testDebugUnitTest
-echo "--- 🧪 Download Unit Test Dependencies [Assemble Unit Tests]"
-./gradlew assembleJalapenoDebugUnitTest assembleDebugUnitTest
-echo ""
-
-# .buildkite/commands/run-instrumented-tests.sh -> build_and_instrumented_test
-#  -> gradle(tasks: %w[assembleVanillaDebug assembleVanillaDebugAndroidTest])
-echo "--- 🧪 Download Android Test Dependencies [Assemble Android Tests]"
-./gradlew assembleJalapenoDebugAndroidTest
+echo "--- 📦 Download Dependencies"
+./gradlew downloadDependencies androidDependencies
```

Comment:
WDYT about adding the task dependency to `downlaodDependencies` so we don't have to remember about running `androidDependencies` every time, and have this as one-task?

I mean something like this in the initialization script

```groovy
tasks.named('downloadDependencies').configure {
    finalizedBy 'androidDependencies'
}
```

and then here

```suggestion
./gradlew downloadDependencies
```

Reply:
Oh, yea, this is a great idea @wzieba , thanks for suggestion! 💯 

Btw, I thought about the pros/cons of doing something like that, for example:

- Cons: Being implicit, not flexible. 🤔
- Pros: When and if need be, we will only update the Android AMI to add/remove extra tasks from this `downloadDependencies`, without the need to open multiple PRs on every client/library project. 🥇 

And I think this single `pros` make this change worthy it. 👍 

FYI: I'll jump right on it, create another staging Android AMI to quickly test this change on this and the JP/WPAndroid#21609 PR, just to be safe, and before proceeding with making all the related PRs ready for review, starting with buildkite-ci#585.

Reply:
This above is now 2f4941b531b68e0b86ae0da77fd6eab9eefbfdd8 ✅ @wzieba and everything is looking as expected:

- Before: [950.32 MB](https://buildkite.com/automattic/woocommerce-android/builds/26803#01949877-fadf-444d-b544-e5a13e487a09/17548-17560) cache size -> [4009](https://scans.gradle.com/s/ffvlzfhqdddko/performance/network-activity) network requests
- After: [950.29 MB](https://buildkite.com/automattic/woocommerce-android/builds/26851#0194ac81-0604-4207-8292-90b8d2eb0da8/17554-17566) cache size -> [4009](https://scans.gradle.com/s/zr2pluvvzemme/performance/network-activity) network requests

Reply:
Thanks! 🙌